yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -156,87 +185,68 @@ func packAndSendAssessMigrationPayload(status string, errMsg string) {
 		}),
 	}
 
-	unsupportedDatatypesList := lo.Map(assessmentReport.UnsupportedDataTypes, func(datatype utils.TableColumnsDataTypes, _ int) string {
-		return datatype.DataType
-	})
+	explanation, err := buildMigrationComplexityExplanation(source.DBType, assessmentReport, "")
+	if err != nil {
+		log.Errorf("failed to build migration complexity explanation: %v", err)
+	}
+
+	var obfuscatedIssues []callhome.AssessmentIssueCallhome
+	for _, issue := range assessmentReport.Issues {
+		obfuscatedIssue := callhome.AssessmentIssueCallhome{
+			Category:              issue.Category,
+			CategoryDescription:   issue.CategoryDescription,
+			Type:                  issue.Type,
+			Name:                  issue.Name,
+			Description:           issue.Description,
+			Impact:                issue.Impact,
+			ObjectType:            issue.ObjectType,
+			ObjectName:            constants.OBFUSCATE_STRING,
```

Comment:
let's remove ObjectName from the struct? We will always obfuscate it, right? 
Even for sqlstatement, maybe in future we will send, but for now, let's just not send it?

Reply:
yeah for object name we will probably always, for sql statement we might start obfuscating and sending specific parts.

Just to be future proof i thought to have it there, and the queries we can ignore these fields for now??

Reply:
will remove ObjectName and SqlStatement field